independent-publisher-connectors/ORB Intelligence/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4731 @@
+                                "format": "int32"
+                              },
+                              "followers": {
+                                "type": "integer",
+                                "format": "int32"
+                              }
+                            },
+                            "description": "URL to the company's Youtube page",
+                            "example": "null"
+                          },
+                          "technologies": {
+                            "type": "array",
+                            "items": {
+                              "type": "object",
+                              "properties": {
+                                "name": {
+                                  "type": "string"
+                                }
+                              }
+                            },
+                            "description": "List of frontend technologies installed on the company's website",
+                            "example": [
+                              {
+                                "name": "Gravatar"
+                              },
+                              {
+                                "name": "WordPress"
+                              }
+                            ]
+                          },
+                          "rankings": {
+                            "type": "array",
+                            "items": {
+                              "type": "string"
+                            },
+                            "description": "Popular list the company may be included, such as Fortune, Forbes, Inc.com",
+                            "example": "null"
+                          },
+                          "ranking_positions": {
+                            "type": "array",
+                            "items": {
+                              "type": "object",
+                              "properties": {
+                                "ranking": {
+                                  "type": "string",
+                                  "example": "Forbes Global 500"
+                                },
+                                "position": {
+                                  "type": "integer",
+                                  "format": "int32",
+                                  "example": 1
+                                }
+                              }
+                            },
+                            "description": "Popular list the company may be included, such as Fortune, Forbes, Inc.com (with positions)"
+                          },
+                          "eins": {
+                            "type": "array",
+                            "items": {
+                              "type": "string"
+                            },
+                            "description": "U.S. Federal Employer Identification Number",
+                            "example": []

```

Comment:
@aaryan2134 Empty properties can be removed until unless they are adding any value to the connector.